### PR TITLE
fix: default keycloak realm envs

### PR DESCRIPTION
**Pull request by chance-coleman**
## Description
Updating default values for keycloak realm settings, identity-config realm.json files now use default values in the realm.json so there is no need for default values to live in uds-core. So removed those default values and instead added a comment block of values that could be used to override the default values.

## Related Issue
[Relates to this uds-identity-config PR](https://github.com/defenseunicorns/uds-identity-config/pull/95)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Other (security config, docs update, etc)

## Checklist before merging

- [x] Test, docs, adr added or updated as needed
- [x] [Contributor Guide Steps](https://github.com/defenseunicorns/uds-template-capability/blob/main/CONTRIBUTING.md)(https://github.com/defenseunicorns/uds-template-capability/blob/main/CONTRIBUTING.md#submitting-a-pull-request) followed

### CODE REVIEW

**Comment by chance-coleman** [comment]
This wont pass CI until [this PR](https://github.com/defenseunicorns/uds-identity-config/pull/95) is merged in uds-identity-config because keycloak is looking for the default env values still.

File: src/keycloak/chart/values.yaml [hunk 1]
```diff
@@ -15,12 +15,20 @@ domain: "###ZARF_VAR_DOMAIN###"
...
 realm: uds
 
-# UDS Identity Config Environment Variables. More info here: https://github.com/defenseunicorns/uds-identity-config/blob/main/docs/CUSTOMIZE.md#templated-realm-values
-realmInitEnv:
-  GOOGLE_IDP_ENABLED: false
-  # Other UDS Identity Config fields that will be used in the realm.json initalization of keycloak
-  # GOOGLE_IDP_CLIENTID: ""
-  # GOOGLE_IDP_CLIENT_SECRET: ""
+# UDS Identity Config Realm Environment Variables. More info here: https://github.com/defenseunicorns/uds-identity-config/blob/main/docs/CUSTOMIZE.md#templated-realm-values
+# realmInitEnv:
```
Comment (mjnagel) [inline src/keycloak/chart/values.yaml]: nit: I'm a fan of `realmInitEnv: {}` (not commented out, but empty by default) in addition to the commented block here. Shouldn't change any templating/functionality but draws more attention to the value existing imo.

**Review by mjnagel — DISMISSED** [review]

**Review by mjnagel — APPROVED** [review]

File: src/keycloak/chart/values.yaml [hunk 2]
```diff
@@ -15,12 +15,20 @@ domain: "###ZARF_VAR_DOMAIN###"
...
-# UDS Identity Config Environment Variables. More info here: https://github.com/defenseunicorns/uds-identity-config/blob/main/docs/CUSTOMIZE.md#templated-realm-values
+# UDS Identity Config Realm Environment Variables. More info here: https://github.com/defenseunicorns/uds-identity-config/blob/main/docs/CUSTOMIZE.md#templated-realm-values
 realmInitEnv:
   GOOGLE_IDP_ENABLED: false
-  # Other UDS Identity Config fields that will be used in the realm.json initalization of keycloak
-  # GOOGLE_IDP_CLIENTID: ""
-  # GOOGLE_IDP_CLIENT_SECRET: ""
+#   GOOGLE_IDP_ID: ""
+#   GOOGLE_IDP_SIGNING_CERT: ""
+#   GOOGLE_IDP_NALE_ID_FORMAT: ""
```
Comment (mjnagel) [inline src/keycloak/chart/values.yaml:23]: Is this supposed to be NAME? Just realized when typing up release announcement...

Comment (chance-coleman) [reply src/keycloak/chart/values.yaml:23]: https://github.com/defenseunicorns/uds-core/pull/462

**Pull request merged**